File: homework_01/task09.py
# task09
import math

def simple_func(n):
    i = 0
    j = 0
    if n % 2 == 0 and not math.floor(n / 10): i += 1
    elif n % 3 == 0 and n % 10 == 1: j += 1
    return [i,j]

for x in range(15):
    print(x, " --> ", simple_func(x))

File: Homeworks/homework_01/archive/task09.js
"use strict";

let n = +prompt();
let i = 0, j = 0;

if ((n % 2 === 0) && (!Math.floor(n / 10))) {
    i++
}
if ((n % 3 === 0) && (n % 10 === 1)) {
    j++
}
console.log(i);
console.log(j);